src/middle/Stan_math_signatures.ml

```diff
@@ -905,6 +906,8 @@ let () =
   add_unqualified
     ("elt_multiply", ReturnType URowVector, [URowVector; URowVector]) ;
   add_unqualified ("elt_multiply", ReturnType UMatrix, [UMatrix; UMatrix]) ;
+  add_unqualified ("elt_multiply", ReturnType UMatrix, [UMatrix; UReal]) ;
+  add_unqualified ("elt_multiply", ReturnType UMatrix, [UReal; UMatrix]) ;
```

Comment:
These will also work with row_vector and vector (matrix, vector and row_vector are eigen types so if it works for matrices it also automatically works for vectors/row_vectors. See the suggestion for divide.